toolchain/check/context.h

```diff
@@ -242,7 +242,7 @@ class Context {
                              bool required = true) -> LookupResult;
 
   // Performs a name lookup in a specified scope, returning the referenced
-  // instruction. Does not look into extended scopes. Returns an invalid
+  // instruction. Does not look into extended scopes. Returns a `None`
   // instruction if the name is not found.
```

Comment:
To revisit after this lands: the phrasing here doesn't seem right. (`None` isn't an instruction.)

Reply:
Revisiting in #4836